docs/source/API/containers/ScatterView.rst

```diff
@@ -0,0 +1,126 @@
+``ScatterView``
+===============
+
+.. role:: cppkokkos(code)
+	:language: cppkokkos
+
+Header File: ``<Kokkos_ScatterView.hpp>``
+
+.. _parallelReduce: ../core/parallel-dispatch/parallel_reduce.html
+
+.. |parallelReduce| replace:: :cpp:func:`parallel_reduce`
+
```

Comment:
Please make sure the sections are consistently ordered from one API entry to another.  The copy - paste-able code bloc (here, called Usage) should be the last section.  Perhaps rename "Usage" "Example," and make it the last section.  "Description" should come after the `Header File` and should be a brief human language description.  Let's move towards standardizing API entries.